css-borders-4/Overview.bs

```diff
@@ -367,45 +367,71 @@ Issue <a href="https://github.com/w3c/csswg-drafts/issues/11610">#11610</a>: che
 
 			Note: this is the initial value of 'corner-shape' properties, as elements with 'border-radius' would be rounded.
 
 		<dt><dfn>scoop</dfn>
-		<dd>Border radii define a concave elliptical curve at the corner. Equivalent to <css>superellipse(0.5)</css>.
+		<dd>Border radii define a concave elliptical curve at the corner. Equivalent to <css>superellipse(-1)</css>.
 		<dt><dfn>bevel</dfn>
-		<dd>Border radii define a diagonal slice at the corner. Equivalent to <css>superellipse(1)</css>.
+		<dd>Border radii define a diagonal slice at the corner. Equivalent to <css>superellipse(0)</css>.
 		<dt><dfn>notch</dfn>
-		<dd>Border radii define a concave 90deg angle at the corner. Equivalent to <css>superellipse(0)</css>.
-		<dt><dfn>straight</dfn>
+		<dd>Border radii define a concave 90deg angle at the corner. Equivalent to <css>superellipse(-infinity)</css>.
+		<dt><dfn>square</dfn>
 		<dd>Border radii define a convex 90deg angle at the corner.
 			This would have the same visual effect as a 'border-radius' of 0. This is different from having a 'border-radius' of 0 when animating.
 			Equivalent to <css>superellipse(infinity)</css>.
 		<dt><dfn>squircle</dfn>
-		<dd>Border radii define a convex curve between an ellipse and an convex angle, equivalent to <css>superellipse(4)</css>.
+		<dd>Border radii define a convex curve between an ellipse and an convex angle, equivalent to <css>superellipse(2)</css>.
 	</dl>
 
-	Issue <a href="https://github.com/w3c/csswg-drafts/issues/11607">#11607</a>: resolve on ''straight'' vs <css>none</css>.
+	The <dfn>superellipse( <<number>> | infinity | -infinity )</dfn> function describes the <dfn>superellipse parameter</dfn> of the corner.
+	It is a number between <css>-infinity</css> and <css>infinity</css>, with <css>-infinity</css> corresponding to a straight concave corner,
+	<css>infinity</css> corresponding to a square convex corner.
 
-	The <dfn>superellipse( <<number>> | infinity )</dfn> function describes the curvature of the corner.
-	It accepts a <dfn>superellipse exponent</dfn>, which defines the curvature of the corner, or the exponent of the formula.
-	The [=superellipse exponent=] accepts values between 0 (a straight concave angle) and <css>infinity</css> (a straight convex angle),
-	with the values in between representing the curvatures in between.
-
-	The ''superellipse()'' formula can be described in cartesian coordinates, as follows,
-	where <code>a</code> is the horizontal ''border-radius''
-	<code>b</code> is the vertical ''border-radius'', and <code>k</code> is the [=superellipse exponent=]:
+	The <dfn>canonical superellipse formula</dfn> can be described in cartesian coordinates, as follows,
+	where <code>s</code> is the [=superellipse parameter=]:
 
 	<pre>
-		|x/a|^k + |y/b|^k = 1
+		k = 2<sup>abs(|s|)</sup>
+		x<sup>k</sup> + y<sup>k</sup> = 1
 	</pre>
 
+	The resulting |x| and |y| are later projected to CSS coordinates by mapping to the corner created by the 'border-radius' properties,
```

Comment:
Does this need to say something about scaling based on `border-radius` ?

Reply:
Reworded